src/kudu/consensus/raft_consensus.cc

```diff
@@ -3586,6 +3586,7 @@ void RaftConsensus::HandleProxyRequest(const ConsensusRequestPB* request,
 
     vector<ReplicateRefPtr> messages;
     do {
+      messages.clear();
```

Comment:
the description of the diff does not describe clearly what was the bug fix and what was the issue?
Is it that under stress we were not clearing the messages and now we had the same messages multiple times and
then we were having an OpId mismatch error below.

Reply:
I saw messages from the previous batch leak into the next batch at the end. It's less likely to be provoked with a slow trickle of updates.